worker:
    evaluation:
        time_limit: 15 #[s]
        number_of_tries: 2
        use_dyn_reconf: True
        dyn_reconf_name: "walking/engine"
        start_position: [0,0, 0.41] #x,y,z
        start_orientation: [0, 0.18,0] #r,p,y
        model_name: "wolfgang"
        link_name: "wolfgang::head"

    simulation:
        sim_time_step: 0.0005
        sim_max_update_rate: 1000
        ode_sor_pgs_precon_iters: 200
        ode_sor_pgs_iters: 200
        ode_sor_pgs_w: 1
        ode_sor_pgs_rms_error_tol: 1
        ode_contact_surface_layer: 1
        ode_contact_max_correcting_vel: 100
        ode_cfm: 0
        ode_erp: 0.2
